libethereum/BlockChain.cpp

```diff
@@ -364,21 +364,21 @@ pair<ImportResult, ImportRoute> BlockChain::attemptImport(bytes const& _block, O
 	}
 	catch (UnknownParent&)
 	{
-		return make_pair(ImportResult::UnknownParent, make_pair(h256s(), h256s()));
+		return make_pair(ImportResult::UnknownParent, ImportRoute());
```

Comment:
Because `ImportRoute()` syntax can be ambiguous in some cases. Like `ImportRoute x();` :) Not exactly the same but `()` are wrong here.

I think we discussed it in the past and decided to prefer `{}` over `()`. If not should we choose one option in coding standards?